docs/markdown/Reference-tables.md

```diff
@@ -1,5 +1,14 @@
 # Reference tables
 
+## Implementations (since 1.3.0)
+
+These are return values of `meson.implementation()` depending on known
+re-implementations.
+
+| Value | Implementation |
+| ----- | -------------- |
+| meson | Meson          |
+
```

Comment:
Nitpicking: I would move that at the bottom of the page since that's much less important information than compiler ids, etc.